report/src/introduction/agile_approach.tex

```diff
@@ -1,29 +1,27 @@
 \section{Implementing Agile Project Management with Scrum}
 Developing a software product for a business can be a complex process.
-Incremental agile approaches have become popular to address this complexity, as they allow for faster development and adaptation to changes.\cite{sommervilleSoftwareEngineering2016}
-In incremental agile approaches, a system is developed as a series of increments, each adding functionality to the system.
-Between these increments, one gathers feedback from users to prioritize functionality for the next increment.
+Incremental agile approaches have become popular to address this complexity, as it allows for fast adaptation to changes in user needs. \cite{sommervilleSoftwareEngineering2016}
```

Comment:
Writing `it` is wrong as the sentence refers to multiple agile approaches.
Narrowing down to user needs is also, in my view, wrong. You learn about the problem as you explore solutions. That doesn't just include users changing needs.

Reply:
I think that saying it allows for faster development is a bit of a stretch though. When I read that I think 'okay, so why is this not something everyone does?'
Maybe by saying 'these methods allows for fast adaptation to rapidly changing user needs and therefore faster development of software products where this is the case.'?